libvuln/opts.go

```diff
@@ -64,9 +64,9 @@ func (o *Opts) Parse() error {
 			&python.Matcher{},
 		}
 	}
-	if len(o.Updaters) == 0 {
+	if len(o.Updaters.Set) == 0 {
```

Comment:
Thinking about airgap support, this needs to be able to express "no updaters". I suppose one would do it presently by providing a regexp that matches none of the defaults' names, but this seems roundabout.

Reply:
This is is the only method that uses the .Set field, and I agree we should probably not export Set, I can give the ```UpdaterSet``` a Length() field, or an Empty() field. I am thinking maybe Length is not that useful?